### ISSUE TITLE
Support re-create a new calculation using a finished job.

### ISSUE THREAD

**Issue by superstar54** [aiidalab/aiidalab-qe#634]
Sometimes, the user wants to create a new calculation using the setting parameters from a finished job and adjust some parameters.

It would be very helpful if we could support it.

**Comment by danielhollas** [comment]
btw: I am doing this by default in my app, by not resetting the workflow setup widgets at all when user presesess the reset button, since this seems like a fairly common use case.

**Comment by superstar54** [comment]
Thanks @danielhollas  for the suggestion.

> by not resetting the workflow setup widgets at all when user presesess the reset button,

Will this confuse the user?

Besides, users may not notice the overrides in advanced settings. Resetting everything is a safe action for the user. I would add a new button, like "Use this calculation as a template"

**Comment by danielhollas** [comment]
Yeah, my observation was that often a user might want to launch the same type of calculation on different molecules, so not resetting the widgets makes sense. But it might be confusing in the QeApp context.

**Comment by edan-bainglass** [comment]
Closed by #1429
